### PR TITLE
DIRECTOR: Preload Stxts data in textCast

### PR DESCRIPTION
**Pull request by tobiatesan**
***EDIT*: I've opened #944, which includes this, which, in turn, includes #942, and adds caching of MacTexts.
If you like all changes please merge that one.**
------------------------------
This is #942, with, additionally, loading of Stxts into textCasts rightaway in Score.

If you like it please merge this one.

Cached MacTexts are coming soon.

### CODE REVIEW

File: engines/director/frame.cpp [hunk 1]
```diff
@@ -948,6 +854,7 @@ void Frame::renderText(Graphics::ManagedSurface &surface, uint16 spriteId, Commo
 	inkBasedBlit(surface, textWithFeatures, spriteId, Common::Rect(x, y, x + width, y + height));
 }
 
+
```
Comment (tobiatesan) [inline engines/director/frame.cpp]: Whoooops

Comment (tobiatesan) [inline engines/director/frame.cpp]: Fixed

**Comment by tobiatesan** [comment]
**I've opened #944, which includes this, which, in turn, includes #942, and adds caching of MacTexts.
If you like all changes please merge that one.**

**Comment by sev-** [comment]
Closing in favor of #944

**Pull request closed without merging**